src/constants/enum.ts

```diff
@@ -34,3 +34,9 @@ export enum MajorSkillId {
   Bx = 8,
   EtcDesign = 9,
 }
+
+export enum LessonLevelId {
+  Top = 1,
+  Middle = 2,
+  Bottom = 3,
+}
```

Comment:
```suggestion
// 과제물 난이도
export enum LessonLevelId {
  // 상
  Top = 1,
  // 중
  Middle = 2,
  // 하
  Bottom = 3,
}
```
처음 enum 또는 상수를 만들때부터 얘기했어야하긴 하는데 영어 네이밍으로 모든 형태를 표현할 수 없어서 주석을 다는데 이런 상수또한 마찬가지로 생각합니다 이런식으로 주석을 달았을 때 해당 파일에 들어가지 않아도 에디터에서 주석을 보여줄 수 있습니다. 다같이 한번 얘기해보면 좋을거같네요.
@RokwonK

Reply:
enum이나 상수를 만들 때 영어로 네이밍을 하면서 설명이 부족하다는 생각을 했었는데 주석으로 부족한 영어 네이밍을 보충하는 방법 좋은 것 같습니다!

Reply:
석호님께서 말씀해 주신 방법이 깔끔한 것 같습니다! 에디터에서 주석을 볼 수 있다니!

Reply:
그러면 앞으로 enum 또는 const 로 상수를 만들 때 주석을 다는 것으로 컨벤션을 가져가죠